unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/GltfContainer/GltfContainerHandler.cs

```diff
@@ -414,7 +414,7 @@ void RemoveActiveColliders(IList<Collider> colliders, IInternalECSComponent<Inte
             if (rendererHashSet == null || rendererHashSet.Count == 0)
                 return;
 
-            var model = renderersComponent.GetFor(scene, entity)?.model ?? new InternalRenderers();
+            var model = renderersComponent.GetFor(scene, entity)?.model ?? new InternalRenderers(new List<Renderer>());
```

Comment:
here as well - is it possible to cache, clear and re-use instead of creating new instances?